python3/utils.py

```diff
@@ -590,15 +590,15 @@ def split_filelist(filelist_string):
 
 def parse_file_search_result_line(line, accession, output_format):
     cols = line.split('\t')
-    data_acc = cols[0].strip()
-    sub_filelist = split_filelist(cols[1])
-    sub_md5list = split_filelist(cols[2])
+    data_acc = cols[-1].strip()
```

Comment:
what does the response look like? can u add an example here as a comment

Reply:
This is the request to the API:
https://www.ebi.ac.uk/ena/portal/api/search?query=sample_accession=%22SAMEA1116773%22&result=read_run&fields=submitted_ftp,submitted_md5,sra_ftp,sra_md5,fastq_ftp,fastq_md5&limit=0

Response:
| submitted_ftp                                                                                                                                                               | submitted_md5                                | sra_ftp                                                       | sra_md5                               | fastq_ftp                                                                                                                | fastq_md5                              | run_accession |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|---------------------------------------------------------------|---------------------------------------|--------------------------------------------------------------------------------------------------------------------------|----------------------------------------|---------------|
| ftp.sra.ebi.ac.uk/vol1/run/ERR251/ERR2512031/20104421_S5_L999_R1_001.fastq.gz;ftp.sra.ebi.ac.uk/vol1/run/ERR251/ERR2512031/20104421_S5_L999_R2_001.fastq.gz | 5267a0aa15395983b08318af330bfe47;e351cea6ed9d2f45f2d5fc01238789e5 | ftp.sra.ebi.ac.uk/vol1/err/ERR251/001/ERR2512031             | 1cf4167dfebec580f3a9f8927c546cc7 | ftp.sra.ebi.ac.uk/vol1/fastq/ERR251/001/ERR2512031/ERR2512031_1.fastq.gz;ftp.sra.ebi.ac.uk/vol1/fastq/ERR251/001/ERR2512031/ERR2512031_2.fastq.gz | 950123b5264f6483040901575a8e8383;8bb209553d1c0292593524813cffb67f | ERR2512031   |

Reply:
submitted_ftp	submitted_md5	sra_ftp	sra_md5	fastq_ftp	fastq_md5	run_accession
ftp.sra.ebi.ac.uk/vol1/run/ERR251/ERR2512031/20104421_S5_L999_R1_001.fastq.gz;ftp.sra.ebi.ac.uk/vol1/run/ERR251/ERR2512031/20104421_S5_L999_R2_001.fastq.gz	5267a0aa15395983b08318af330bfe47;e351cea6ed9d2f45f2d5fc01238789e5	ftp.sra.ebi.ac.uk/vol1/err/ERR251/001/ERR2512031	1cf4167dfebec580f3a9f8927c546cc7	ftp.sra.ebi.ac.uk/vol1/fastq/ERR251/001/ERR2512031/ERR2512031_1.fastq.gz;ftp.sra.ebi.ac.uk/vol1/fastq/ERR251/001/ERR2512031/ERR2512031_2.fastq.gz	950123b5264f6483040901575a8e8383;8bb209553d1c0292593524813cffb67f	ERR2512031

Reply:
What happened is that the accession came first in the response until now, but it changed to being the last.

Reply:
right I meant add an this in the code file as a comment so the column numbers for the values make sense